unit/Makefile

```diff
@@ -22,9 +22,11 @@ SRC += analyses/ai/ai.cpp \
        goto-programs/goto_program_declaration.cpp \
        goto-programs/goto_program_function_call.cpp \
        goto-programs/goto_program_goto_target.cpp \
+       goto-programs/goto_program_instruction.cpp \
        goto-programs/goto_program_symbol_type_table_consistency.cpp \
        goto-programs/goto_program_table_consistency.cpp \
        goto-programs/goto_trace_output.cpp \
+       goto-programs/goto_program_validate.cpp \
```

Comment:
Should be moved up by one line to maintain sorting.

Reply:
Ach! Done.